contracts/src/instances/rseth-linea/RsETHLineaConversions.sol

```diff
@@ -29,7 +29,7 @@ library RsETHLineaConversions {
     function convertToBase(
         IRSETHPoolV2 _rsETHPool,
         uint256 _shareAmount
-    ) external view returns (uint256) {
+    ) internal view returns (uint256) {
```

Comment:
I noticed that these were `external` during a test deployment. I switched this to `internal` to decrease gas costs, and the code size was not an issue. That